pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.black]
+line-length = 88
+target-version = ['py37']
```

Comment:
main build version is declared as 3.8 in the above?

Reply:
i don't think it matters that the build is done on py3.8 but we ask `black` to use py3.7. The two are separate concerns.

Reply:
ok, wasn't sure if `black` minded.